src/inference/src/infer_request.cpp

```diff
@@ -267,8 +262,6 @@ bool InferRequest::wait_for(const std::chrono::milliseconds timeout) {
     OPENVINO_SUPPRESS_DEPRECATED_START
     try {
         return _impl->wait_for(timeout);
-    } catch (const InferenceEngine::InferCancelled& e) {
-        Cancelled::create(e.what());
```

Comment:
I suppose we need to catch `ov::Cancelled` here and re-throw as in `ov::InferRequest::wait`